api/src/certification/session/application/session-mass-import-controller.js

```diff
@@ -0,0 +1,47 @@
+import { usecases } from '../../../../lib/domain/usecases/index.js';
+import { usecases as sessionUsecases } from '../../../../src/certification/shared/domain/usecases/index.js';
+
+import * as csvHelpers from '../../../../lib/application/certification-centers/csvHelpers.js';
+import * as csvSerializer from '../../../../lib/infrastructure/serializers/csv/csv-serializer.js';
+
+const validateSessionsForMassImport = async function (request, h, dependencies = { csvHelpers, csvSerializer }) {
```

Comment:
comme on est dans le mass-import-controller :) 
```suggestion
const validateSessions = async function (request, h, dependencies = { csvHelpers, csvSerializer }) {
```